roles/rsyslog/README.md

```diff
@@ -185,8 +185,47 @@ rsyslog_user: root
 
 rsyslog_example
 ---------------
-Once ansible-playbook is run with rsyslog_example: true and rsyslog_pki: true, the following configuration files will be deployed.
+The configuration generated by rsyslog_example basically reads logs from the local unix socket and outputs to the local files, e.g., in /var/log.  This is the minimum variables to set it up.
+```
+rsyslog_enabled: true
+# install example packages & config files
+rsyslog_example: true
+rsyslog_group: root
+rsyslog_user: root
+```
+In addition, rsyslog_example allows to add remote input/output as well as the security over the network.
+
+To redirect the logs to the remote rsyslog servers using the rsyslog rule, you could add rsyslog_forward to the above basic variables.  E.g.,
+```
+rsyslog_forward: [ '*.crit @@10.10.10.1:514', '*.info @@10.10.10.11:514' ]
+```
+where @@ means the logs are sent over tcp; if there is one @, they are sent over udp.
 
+To receive remote input, you could add 'network' to rsyslog_capabilities to the basic variables as follows.
+```
+rsyslog_capabilities: [ 'network' ]
+```
+It will configure imudp as well as imptcp.
+
+To put logs from the remote input in the separate files, e.g., /var/log/remote/hosts/%HOSTNAME%/*.log, 'remote-files' could be added to rsyslog_capabilities.
+```
+rsyslog_capabilities: [ 'network', 'network' ]
```

Comment:
You have a bug here ...
Should be 
rsyslog_capabilities: [ 'network', 'remote-files' ]

@nhosoi